coreos-base/afterburn/files/coreos-metadata-sshkeys@.service

```diff
@@ -12,3 +12,4 @@ ExecStart=/usr/bin/coreos-metadata ${COREOS_METADATA_OPT_PROVIDER} --ssh-keys=%i
 [Install]
 DefaultInstance=core
 RequiredBy=multi-user.target
+Alias=afterburn-sshkeys@.service
```

Comment:
I think we also need an ExecStartPost action here to append the created file in `authorized_keys.d/` to `authorized_keys`, now that update-ssh-keys is not used but the file is simply dropped inside the folder.

Reply:
(or you restore the full cl-legacy features with calling update-ssh-keys)

Reply:
Oh ok understood. To me `update-ssh-keys` was just a lib to write into `./authorized_keys_d/...` and afterburn was doing the merge later.
Indeed things happen there: https://github.com/flatcar-linux/update-ssh-keys/blob/012bc8d9d86f044b0c87ca749563879eefbde25c/src/lib.rs#L306-L309 and there: https://github.com/flatcar-linux/afterburn/blob/bb3cdd0b895e9803c843d1e1831f9c0613e35e02/src/providers/mod.rs#L90-L92